go/sqltypes/arithmetic.go

```diff
@@ -282,13 +284,15 @@ func newIntegralNumeric(v Value) (result numeric, err error) {
 	case v.IsSigned():
 		result.ival, err = strconv.ParseInt(str, 10, 64)
 		if err != nil {
+			err = vterrors.Errorf(vtrpcpb.Code_INVALID_ARGUMENT, "%v", err)
```

Comment:
Internal tools show that this line and line 295 is currently not covered by tests.

Can you please add cases in "TestNewIntegralNumeric" in arithmetic_test.go?

Reply:
Are you sure about that? I ran coverage against this branch and it shows it is covered. Maybe you ran prior to one of my recent commits, which changed the wording of the error? These two lines are covered by the two tests here https://github.com/youtube/vitess/blob/master/go/sqltypes/arithmetic_test.go#L546

The test does not validate the vtrpcpb code, just the error string. If you want me to validate the code I can refactor the test I guess